doc/source/data/performance-tips.rst

```diff
@@ -187,5 +187,19 @@ To enable deterministic execution, set the preceding to True. This setting may d
 Monitoring your application
 ---------------------------
 
-View the Ray Dashboard to monitor your application and troubleshoot issues. To learn
-more about the Ray dashboard, see :ref:`Ray Dashboard <observability-getting-started>`.
+View the Ray Data dashboard located in the Metrics tab of the Ray Dashboard to monitor your application and troubleshoot issues. Ray Data emits Prometheus metrics in real-time while a Dataset is executing, and the Ray Data dashboard displays these metrics grouped by Dataset. Datasets can also be assigned a name using :meth:`~ray.data.Dataset._set_name`, which prefixes the dataset ID for a more identifiable label.
+
+The metrics recorded are:
+
+* Bytes Spilled
+* Bytes Allocated
+* Bytes Freed
+* Object Store Memory
+* CPUs (logical slots)
+* GPUs (logical slots)
+* Bytes Outputted
```

Comment:
can we be more verbose for the metrics?

e.g.
`Bytes Spilled` -> `Objects in Bytes Spilled from Object Store to Disk`